packages/services/src/kernel/default.ts

```diff
@@ -82,6 +84,29 @@ export class KernelConnection implements Kernel.IKernelConnection {
    */
   readonly handleComms: boolean;
 
+  /**
+   * Whether comm messages should be sent to kernel subshells, if the
+   * kernel supports it.
+   *
+   * #### Notes
+   * Sending comm messages over subshells allows processing comms whilst
+   * processing execute-request on the "main shell". This prevents blocking
+   * comm processing.
+   * If enabled, we'll create one subshell per-comm, this may lead to issue
```

Comment:
This text can be updated to also mention one subshell per comm target, and that that is the default.